keps/sig-network/2595-expanded-dns-config/README.md

```diff
@@ -384,6 +384,8 @@ merged](https://github.com/kubernetes/kubernetes/pull/100651)
 merged](https://github.com/kubernetes/website/pull/28096)
 - 2022-01-12: [Docs updated to add requirements for the
 feature](https://github.com/kubernetes/website/pull/31305)
+- 2022-08-15: [Beta implementation
```

Comment:
can you update the testing section to add links to the new e2e tests which suppose to be green and code coverage numbers for unit tests.

Reply:
thanks, done.